src/frameworks/opengles.rs

```diff
@@ -28,11 +28,16 @@
 //!   - [EXT_framebuffer_object](https://registry.khronos.org/OpenGL/extensions/EXT/EXT_framebuffer_object.txt)
 
 pub mod eagl;
+mod gles1_native;
 mod gles1_on_gl2;
 mod gles_generic;
 mod gles_guest;
 
+#[cfg(target_os = "android")]
```

Comment:
Hmm, I'd like it if native OpenGL ES is available as an option on desktop OSes that support it, so can we remove the conditional compilation here? Linux with Mesa drivers supports it so it should just work on that platform. Apparently, some vendors' drivers on Windows might have weird extensions that also provide this…

Reply:
Done
(but I need to add unused_imports to silence a warning)

Reply:
Oh, because the cfg check still exists in other places? I was imagining you could remove all the `cfg(target_os = "android")` checks related to GLES1Native. Instead of always using GLES1Native on Android and GLES1OnGL2 on desktop, we can select dynamically. But given that GLES1OnGL2 is more mature, we should make it the default for now. (Ask me on Discord if you want me to elaborate. Also, if it's more annoying than expected, maybe we can skip it for now.)

Reply:
Actually never mind, considering the presentRenderbuffer thing (below), this isn't going to be practical for now. You can put back the cfg check.

Reply:
Done 😃